frontend/src/utilities/markdown.ts

```diff
@@ -0,0 +1,49 @@
+import sanitizeHtml from 'sanitize-html';
```

Comment:
i've found before that this package to increase bundle size by quite a bit.
I've switched it out for dompurify in quick starts, maybe something to look into if interested:
https://github.com/cloudmosaic/quickstarts/blob/a1a4170b122ea18b85ac38ef9570989a5a3c5e7d/packages/module/src/ConsoleInternal/components/markdown-view.tsx#L8